### PR TITLE
Polish readme + contributing + licence docs

### PR DESCRIPTION
**Pull request by susodapop**
## Description

- Update README
- Add NOTICE
- Add CONTRIBUTING.md

**Important**: I don't have context on the project layout, formatting, or testing requirements. These are left as **TBD**.

### CODE REVIEW

**Review by moderakh — APPROVED** [review]
LGTM. thanks @susodapop

**Comment by moderakh** [comment]
@susodapop can we merge this? IMO it is fine to have TBD in Contributing.md for now as is.

**Comment by susodapop** [comment]
I’m hoping to get Arik’s eyes on it re: my changes about unit tests.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+To contribute to this repository, fork it and send pull requests.
+
+## Project structure
+
+TBD
+
+## Run the project
+
+TBD
+```
```
Comment (arikfr) [inline CONTRIBUTING.md:12]: seems orphaned?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Commit messages
+
+TBD
+
+## Pull Request Process
+
+1. Update the [README.md](README.md) or similar documentation with details of changes you wish to make, if applicable.
```
Comment (arikfr) [inline CONTRIBUTING.md:20]: ?

Comment (susodapop) [reply CONTRIBUTING.md:20]: This should say CHANGELOG instead I think. But there's no changelog yet.

Comment (susodapop) [reply CONTRIBUTING.md:20]: I'll add to this PR.

Comment (arikfr) [reply CONTRIBUTING.md:20]: Then we can create one based on the details in the releases page.

File: CONTRIBUTING.md (hunk 1 above)
Comment (susodapop) [reply CONTRIBUTING.md]: I removed it.

File: CONTRIBUTING.md (hunk 2 above)
Comment (susodapop) [reply CONTRIBUTING.md]: I added a changelog file and updated the contributing doc accordingly.

**Review by arikfr — APPROVED** [review]
LGTM

**Pull request merged**